app/services/messaging/twilio/reminder_sms.rb

```diff
@@ -0,0 +1,21 @@
+class Messaging::Twilio::ReminderSms < Messaging::Twilio::Api
+  TWILIO_TEST_SMS_NUMBER = "+15005550006"
+
+  def communication_type
+    Communication.communication_types[:sms]
+  end
+
+  def sender_number
```

Comment:
We _could_ cache this so you don't get unexpectedly changing numbers for a single instance every time you call it. Sampling per instance should be sufficient.